### PR TITLE
fix: fix bug in cluster/slot_set

### PR DESCRIPTION
**Pull request by BorysTheDev**
Added missed argument to the function call.
Fix flush slots functionality for migrations slots
Added tests

### CODE REVIEW

**Comment by kostasrim** [comment]
Can you plz add a description of what the bug was and how it's resolved?

**Review by dranikpg — APPROVED** [review]
It's pretty evident what the bug was 😆 That's quite a number of tests for this simple error 😵

**Comment by BorysTheDev** [comment]
> It's pretty evident what the bug was 😆 That's quite a number of tests for this simple error 😵

I was so upset regarding such stupid bug so write 100% coverage tests for slot_set class

**Review by chakaz — DISMISSED** [review]

File: tests/dragonfly/cluster_test.py [hunk 1]
```diff
@@ -1334,6 +1334,7 @@ async def test_cluster_config_reapply(df_local_factory: DflyInstanceFactory):
     await close_clients(*[node.client for node in nodes], *[node.admin_client for node in nodes])
 
 
+# @pytest.mark.skip(reason="The test is broken")
```
[the remark this answers is not in the document]
Comment (BorysTheDev) [reply tests/dragonfly/cluster_test.py:1337]: I missed that I commented it out)) it's not intentional

**Review by chakaz — DISMISSED** [review]

**Comment by BorysTheDev** [comment]
The problem was bigger than I think and this bug shows a couple more)

**Review by dranikpg — APPROVED** [review]
Maybe worth describing the extended bug now that you found another one 🙂

**Pull request merged**